yarn.lock

```diff
@@ -2,11 +2,994 @@
 # yarn lockfile v1
 
 
+"@babel/code-frame@^7.0.0":
```

Comment:
I also don't understand why all these changes were made to the lock file.  Shouldn't a peer dependency not affect this lock file at all?

Reply:
Hi I ran the command 
```
yarn add lerna@3.x -P
```
I've double-checked that I didn't install it by mistake as "dev" - so I've cloned the repo again, ran this command and got the same result.